ui/analyse/src/explorer/explorerConfig.ts

```diff
@@ -98,9 +100,11 @@ export function view(ctrl: ExplorerConfigCtrl): VNode[] {
               'div.choices',
               d.rating.available.map(function (r) {
                 return h(
-                  'span',
+                  'button',
                   {
-                    class: { selected: d.rating.selected().includes(r) },
+                    attrs: {
+                      'aria-pressed': `${d.rating.selected().includes(r)}`,
+                    },
```

Comment:
doesn't span->button and the removal of the selected class change the way it's displayed in the visual UI?

Reply:
The CSS changes take care of that. Aside from changing selectors, the CSS needed one extra `background: none`. After that, the old and new versions become a pixel-perfect match.

Reply:
Probably a good idea to give it an `@extend %button-none` for good measure which also takes care of the `background: none`.